src/service/service.py

```diff
@@ -166,10 +166,24 @@ def new_service_from_cfg(
 
 class ServiceMng:
 
-    def __init__(self, cli_flags: dict[str, bool], configMng: ConfigMng):
+    def __init__(
+        self,
+        cli_flags: dict[str, bool],
+        configMng: ConfigMng,
+        svcFactory: ServiceFactory,
+    ):
         self.cli_flags = cli_flags
         self.configMng = configMng
-        pass
+        self.svcFactory = svcFactory
+
+    def get_service(self, env_tag: str, svc_tag: str) -> Optional[Service]:
```

Comment:
This method could be written in the following way:

```
def get_service(self, env_tag: str, svc_tag: str) -> Optional[Service]:
    """Get a service by environment tag and service tag."""
    if (envCfg := self.configMng.get_environment(env_tag)) and \
       (svcCfg := envCfg.get_service(svc_tag)):
        return self.svcFactory.new_service_from_cfg(envCfg, svcCfg)
    else
        return None
```

and this would avoid the double nested if and make clode a bit more clear, what do you think?